python/torch_mlir/dialects/torch/importer/jit_ir/build_tools/shape_lib_gen.py

```diff
@@ -707,6 +707,9 @@ def aten〇maximum(self: List[int], other: List[int]) -> List[int]:
 def aten〇bitwise_and〇Tensor(self: List[int], other: List[int]) -> List[int]:
     return upstream_shape_helpers.broadcast(self, other)
 
+def aten〇logical_or(self: List[int], other: List[int]) -> List[int]:
+    return upstream_shape_helpers.broadcast(self)
```

Comment:
This is not right. `broadcast` should take both `self` and `other` as input, so that it can broadcast the smaller smaller tensor to the larger one. Can you add a test for this?